CTFd/__init__.py

```diff
@@ -149,7 +149,13 @@ def create_app(config="CTFd.config.Config"):
         # Alembic sqlite support is lacking so we should just create_all anyway
         if url.drivername.startswith("sqlite"):
             db.create_all()
-            stamp()
+            # stamp call needs the migrations directory passed to it.
+            migrations_dir = utils.__file__.split("CTFd")[0] + "CTFd/migrations"
+            if os.path.isdir(migrations_dir):
```

Comment:
I don't think you actually need this if statement. 

The definition for `migrations_dir` is quite ugly. I think you could probably cut it out entirely and just do something with stamp() and `app.root_path`. Also you should use `os.path.join`.

Reply:
I agree it was quite ugly, at work I'm unable to push to github because of the corporate network, so I was editing directly in the GitHub editor. I'll look into those other options today.